docs/changelog/128854.yaml

```diff
@@ -0,0 +1,11 @@
+pr: 128854
+summary: Mark token pruning for sparse vector as GA
+area: Relevance
+type: docs
```

Comment:
I think we want to change this to `feature` not `docs` as it's a release highlight. See https://github.com/elastic/elasticsearch/pull/125302 as an example